nixos/modules/config/sysctl.nix

```diff
@@ -1,8 +1,23 @@
 { config, lib, ... }:
 
-with lib;
-
 let
+  inherit (lib)
+    concatStrings
+    filterOverrides
+    foldl
+    isBool
+    isInt
+    isString
+    literalExpression
+    mapAttrsToList
+    mdDoc
+    mergeOneOption
+    mkDefault
+    mkOption
+    mkOptionType
+    optionalString
+    types
+  ;
```

Comment:
I agree with most other changes but adding those long lists which need to maintained is IMO not an improvement and much more the opposite. I will always prefer writing lib in front of a function over maintaining this list. I would suggest to leave this file as it is to unblock all other changes where the improvement is clearly visible and not as discutable IMO.